lang/en/tool_excimer.php

```diff
@@ -52,6 +52,8 @@
 $string['auto_settings_desc'] = 'Settings related to automatic profiling.';
 $string['enable_auto'] = 'Enable auto profiling';
 $string['enable_auto_desc'] = 'Any page will be automatically profiled if they exceed the miniumum duration.';
+$string['enable_fuzzy_count'] = 'Enable fuzzy counting';
+$string['enable_fuzzy_count_desc'] = 'This will cause the plugin to maintain an approximate count of page runs using the {$a}. Automatic profiling must also be enabled.';
```

Comment:
Since lang strings are being here, could we also bump the version.php?

Reply:
done